mobile/mobile_test.go

```diff
@@ -748,6 +747,14 @@ func TestMobile_SetUsername(t *testing.T) {
 	}
 }
 
+func TestMobile_SetAvatar(t *testing.T) {
+	err := mobile1.SetAvatar("../mill/testdata/image.jpeg")
```

Comment:
Just takes a path. Though you can also add any IPFS hash, or an existing file hash that's needs decryption.